lisp/transient.el

```diff
@@ -4119,7 +4119,9 @@ we stop there."
               'face 'transient-value))
 
 (cl-defmethod transient-prompt ((obj transient-lisp-variable))
-  (format "Set %s: " (oref obj variable)))
+  (or (and (slot-boundp obj 'prompt)
+           (oref obj prompt) (cl-call-next-method obj))
+      (format "Set %s: " (oref obj variable))))
```

Comment:
Maybe that's just because you didn't put it on a separate line, but at first sight I thought to call to `cl-call-next-method` was the result of an edit error. I think the following is better as it avoids such ambiguities. Otherwise we are left wondering whether `cl-call-next-method` may return `nil`.

```elisp
(cl-defmethod transient-prompt ((obj transient-lisp-variable))
  (if (and (slot-boundp obj 'prompt)
           (oref obj prompt))
      (cl-call-next-method obj)
    (format "Set %s: " (oref obj variable))))
```

Reply:
Before noticing this, I had already pimped the commit message to:

```
transient-prompt(lisp-variable): Fallback to generic prompt

Teach `transient-prompt' to fall back to a generic prompt for
`transient-lisp-variable'.  If a prompt is explicitly provided, then
continue to use that.

(The method for `transient-infix', which is currently being used for
all other types, already falls back to a generic prompt in most cases,
but may in some cases use a prompt that warns that a prompt should be
given explicitly.)
```

but I am not pushing that until you had a chance to see the above.

Reply:
Thanks for suggestions.  Applied.